kuma/static/styles/components/wiki/customcss.scss

```diff
@@ -19,227 +19,12 @@ eventually emptying this file.
     font-size: 60%;
 }
 
-/* Make the 2nd breadcrumbs disappear, only if the page has been regenerated with a modern template */
-.column-container #wikiArticle .themeUpdated {
-    display: none;
-}
-
-/* Used to wrap columns that should all be the same height */
-.equalColumnHeights {
-    display: table;
-    border-collapse: separate;
-    border-spacing: 20px;
-}
-
-.equalColumnHeights div {
-    display: table-cell;
-    width: 50%;
-    height: 100%;
-}
-
-/* Use for large table (.thirdColumn + .***Bg) */
-table#fullwidth-table td.thirdColumn {
-    width: 30%;
-}
-
-.greenBg {
-    width: 30%;
-    background-color: #80ff80;
-}
-
-.lightGreenBg {
-    background-color: #dcffdc;
-}
-
-.yellowBg {
-    background-color: #ffffdc;
-}
-
-.orangeBg {
-    background-color: #ffa064;
-}
-
-.redBg {
-    background-color: #f66;
-}
-
-/* highlighted (bottom) link in call-out boxes */
-.callout-link-container {
-    position: relative;
-    height: 100%;
-}
-
-.callout-link-wrapper {
-    padding-top: 32px;
-}
-
-.callout-link {
-    font-style: italic;
-    font-size: 14px;
-    font-weight: normal;
-    display: block;
-    position: absolute;
-    bottom: 0;
-    right: 0;
-}
-
 /* remove left margin from Font Awesome icons inside wiki
      content; this is messing up document content */
 #wikiArticle #{$selector-icon} {
     margin-left: 0;
 }
 
-/* Iframe for live sample centered (Test) */
-.centered iframe {
-    display: block;
-    margin: 0 auto;
-    border: none;
-}
-
-/* Simulate two columns for landing page, if enough space is available    */
-@media #{$media-query-small-desktop-and-up} {
-    .landing {
-        display: table-row;
-    }
-}
-
-/* For Landing page boxes; these are boxes displayed within columns on landing
-     pages, such as the Community and Related boxes. */
-.landingPageBox {
-    width: 100%;
-    padding: 0;
-    border-bottom: 1px solid #bbb;
-    margin-bottom: 3px;
-}
-
-.landingPageBox ul {
-    padding-left: 44px;
-}
-
-/* Wide column box */
-.wideColumnBox {
-    @include vendorize(column-width, 375px);
-}
-
-.wideColumnBoxList li {
-    line-height: 1.5;
-    list-style-type: none;
-    padding-left: 0;
-    page-break-inside: avoid;
-}
```

Comment:
These two are still used by:

* https://developer.mozilla.org/en-US/docs/MDN/Doc_status/JavaScript
* https://developer.mozilla.org/en-US/docs/MDN/Doc_status/CSS
* https://developer.mozilla.org/en-US/docs/MDN/Doc_status/SVG

`wideColumnBoxList` does not cause much breakage but, removing `wideColumnBox` does.